simuk/tests/test_sbc.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import pymc as pm
+
+import simuk
+
+data = np.array([28.0, 8.0, -3.0, 7.0, -1.0, 1.0, 18.0, 12.0])
+sigma = np.array([15.0, 10.0, 16.0, 11.0, 9.0, 11.0, 10.0, 18.0])
+
+with pm.Model() as centered_eight:
+    obs = pm.MutableData("obs", data)
+    mu = pm.Normal("mu", mu=0, sigma=5)
+    tau = pm.HalfCauchy("tau", beta=5)
+    theta = pm.Normal("theta", mu=mu, sigma=tau, shape=8)
+    y_obs = pm.Normal("y", mu=theta, sigma=sigma, observed=obs)
+
+
+def test_sbc():
```

Comment:
We should also test Bambi, something like this 


np.random.seed(1234)
x = np.random.normal(0, 1, 200)
y = 2 + np.random.normal(x, 1)
df = pd.DataFrame({"x": x, "y": y})
model = bmb.Model("y ~ x", df)
sbc = SBC(model,
        num_simulations=10,
        sample_kwargs={'draws': 25, 'tune': 50})

sbc.run_simulations()
sbc.plot_results(kind="hist");  # maybe also test "hist" plot

Reply:
Have added a check for bambi as well as hist